src/yardstick/cli/explore/image_labels/label_manager.py

```diff
@@ -139,10 +139,11 @@ def match_select_entries(self) -> List[MatchSelectEntry]:
             entries: List[MatchSelectEntry] = []
 
             for match in all_matches:
-                ty = dig(match.fullentry, "artifact", "type", default=dig(match.fullentry, "package_type", default=None))
+                t = tools[match.config.tool_name]
```

Comment:
nit: could be nice to provide a helper function to normalize the tool name and help users to get the tool object, e.g. `tools.get(name=tool_name)`, where normalizing would entail at least case insensitive fetching.